tests/falco/commands_test.go

```diff
@@ -132,3 +134,38 @@ func TestFalco_Cmd_PluginInfo(t *testing.T) {
 			`No suggested open params available.*`),
 		res.Stdout())
 }
+
+func TestFalco_Print_IgnoredEvents(t *testing.T) {
+	t.Parallel()
+	checkDefaultConfig(t)
+	type Data struct {
+		Events []string `json:"events"`
+	}
+	evntfile, err := os.Open("events.json")
+	if err != nil {
+		panic(err)
+	}
+	defer evntfile.Close()
+
+	var events Data
+	err = json.NewDecoder(evntfile).Decode(&events)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Look at the comment below: I think this should be part of the test data package under `tests/data/outputs`, so that it could be accessible by other tests as well and reused.